script/build

```diff
@@ -52,34 +51,34 @@ def main():
 
   os.chdir(SOURCE_ROOT)
 
-  for component in COMPONENTS:
-    if args.component == 'all' or args.component == component \
-        or (args.include_ffmpeg and component == 'ffmpeg'):
-      if component == 'shared_library' and args.no_shared_library:
-        continue
-      elif component == 'static_library' and args.no_static_library:
-        continue
-      out_dir = get_output_dir(SOURCE_ROOT, target_arch, component)
-      target = 'chromiumcontent:chromiumcontent'
-      subprocess.check_call([NINJA, '-C', os.path.relpath(out_dir), target], env=env)
-      if component == 'static_library':
-        subenv = env.copy()
-        subenv['CHROMIUMCONTENT_2ND_PASS'] = '1'
-        target = 'chromiumcontent:libs'
-        subprocess.check_call([NINJA, '-C', os.path.relpath(out_dir), target], env=subenv)
+  # Build everything if a components list is non provided.
+  components_to_build = args.component if args.component is not None else COMPONENTS
```

Comment:
It seems like this is mutually exclusive with `args.debug` and `args.release`, so here's a small suggestion to improve readability:

```python
if args.debug:
  components_to_build = ['shared_library', 'ffmpeg']
elif args.release:
  components_to_build = ['static_library', 'ffmpeg']
else:
  # debug/release not specified, build the specified component or "all"
  components_to_build = args.component if args.component is not None else COMPONENTS
```

And in parse args:

```python
group = parser.add_mutually_exclusive_group()
group.add_argument('-c', '--component', nargs='+', default=None, choices=COMPONENTS)
group.add_argument('-D', '--debug', default=False, action='store_true', help='Build debug configuration')
group.add_argument('-R', '--release', default=False, action='store_true', help='Build release configuration')
```

Reply:
Makes sense. Thanks.